explore/src/main/scala/explore/targets/TargetSelectionPopup.scala

```diff
@@ -0,0 +1,220 @@
+            header = ModalHeader("Search Target"),
+            content = ModalContent(
+              <.span(ExploreStyles.TargetSearchTop)(
+                <.span(ExploreStyles.TargetSearchInput)(
+                  FormInputEV(
+                    id = NonEmptyString("name"),
+                    value = inputValue,
+                    // TODO Investigate if we can replicate SUI's "input with icon" styles (which use <i>) but using <svg>,
+                    // so that they work with fontawesome.
+                    // icon = Icons.Search,
+                    // iconPosition = IconPosition.Left,
+                    onTextChange = t => inputValue.setState(t) >> timer.submit(search(t)).runAsync,
+                    loading = searching.value.nonEmpty
+                  )
+                    .withMods(^.placeholder := "Name", ^.autoFocus := true)
+                ),
+                <.div(ExploreStyles.TargetSearchPreview)(
+                  selectedTarget.value
+                    .map(_.target)
+                    .collect { case SiderealTarget(_, tracking, _) =>
+                      tracking.baseCoordinates
+                    }
+                    .map(coordinates =>
+                      Aladin.component
+                        .withRef(aladinRef)
+                        .withKey(
+                          selectedTarget.value.foldMap(t => s"${t.sourceIndex}-${t.resultIndex}")
+                        )(
+                          Aladin(
+                            ExploreStyles.TargetSearchAladin,
+                            showReticle = false,
+                            showLayersControl = false,
+                            target = Coordinates.fromHmsDms.reverseGet(coordinates),
+                            fov = 0.25,
+                            showGotoControl = false
+                          )
+                        )
+                    )
+                    .whenDefined
+                )
+              ),
+              SegmentGroup(raised = true, clazz = ExploreStyles.TargetSearchResults)(
+                results.value.map { case (source, (sourceIndex, targets)) =>
+                  Segment(
+                    <.div(
+                      Header(size = Small)(
+                        s"${source.name} (${showCount(targets.length, "result")})"
+                      ),
+                      <.div(ExploreStyles.TargetSearchResultsSource)(
+                        TargetSelectionTable(
+                          targets.toList,
+                          onSelected = props.onSelected.map(onSelected =>
+                            t => onSelected(t) >> isOpen.setState(false) >> cleanState
+                          ),
+                          selectedIndex = selectedTarget.value
+                            .filter(_.sourceIndex === sourceIndex)
+                            .map(_.resultIndex),
+                          onClick = Reuse.always { case (target: Target, index: Int) =>
```

Comment:
How is this being used? It is not wrapped in a component

Reply:
I'm not sure I undestand the question.